### PR TITLE
UTOPIA-782: Minio Deployment Templates

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

UTOPIA-782: Minio Deployment Templates
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: openshift/minio/minio-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+              initialDelaySeconds: 120
+              periodSeconds: 20
+            env:
+              - name: MINIO_ROOT_USER
+                value: ${MINIO_ROOT_USER}
+              - name: MINIO_ROOT_PASSWORD
+                value: ${MINIO_ROOT_PASSWORD}
+            resources:
+              requests:
+                memory: 2Gi
```
Comment (will-freshworks) [inline openshift/minio/minio-deployment.yaml:49]: That's a lot of memory. Have you confirmed that it has this heavyweight a footprint?

**Review by will-freshworks — APPROVED** [review]
Looks simple enough

File: openshift/minio/minio-deployment.yaml (hunk 1 above)
Comment (akroon3r) [reply openshift/minio/minio-deployment.yaml:49]: I can tune this backdown NBD, but when I first spun it up, the response to the URL was really slow.

I turned both memory and CPU up from the values that I found were in this template originally and the console seemed to respond a bit more quickly.

Whatever you think is best, I can adjust NBD.

**Comment by will-freshworks** [comment]
This is fine for now to build against but I think this would be the best approach for production: https://github.com/kubernetes/examples/blob/master/staging/storage/minio/minio-distributed-statefulset.yaml

**Pull request closed without merging**